app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/location/LocationArmoryActivity.java

```diff
@@ -62,7 +71,11 @@ public void completeFireEvent(View view){
      */
     public void completeHostileShipEvent(View view){
         if(ship.getEventLog().contains(Events.HOSTILE)){
-            game.sendMessage(new EventCompletedMessage("Hostile Ship Event", ship.getEventLog().pop(Events.HOSTILE).getId()));
+            EventCompletedMessage message = new EventCompletedMessage("Hostile Ship Event", ship.getEventLog().pop(Events.HOSTILE).getId());
+            Map<String, Integer> parameters = new HashMap<String, Integer>();
+            parameters.put("testParam", 42);
```

Comment:
Just hardcoded for testing purposes, this is also the parameter that the server generates hardcoded for the Event currently. Of course these must become the parameters entered in the user interface, but that is to be done during minigame implementation.